Exception was thrown when two boostnote is launched

# Current behavior
I tried to launch Boostnote as another process with Boostnote already running, then the following exception was thrown:
```
(electron) 'app.makeSingleInstance(cb)' is deprecated. Use 'app.requestSingleInstanceLock() and app.on('second-instance', cb)' instead.
internal/modules/cjs/loader.js:604
    throw err;
    ^

Error: Cannot find module '../dialog'
    at Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
    at Function.Module._resolveFilename (/usr/share/boostnote/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (internal/modules/cjs/loader.js:528:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.get [as dialog] (/usr/share/boostnote/resources/electron.asar/browser/api/exports/electron.js:11:16)
    at process.<anonymous> (/usr/share/boostnote/resources/electron.asar/browser/init.js:54:31)
    at process.emit (events.js:182:13)
    at process._fatalException (internal/bootstrap/node.js:451:27)
```

# Expected behavior
If Boostnote is started when Boostnote is already running, above exception should not be thrown.

# Steps to reproduce

<!-- 
Please be thorough, issues we can reproduce are easier to fix!
-->

1. launch Boostnote
2. launch Boostnote as another process

# Environment

- Version : 0.12.1
- OS Version and name : Arch Linux

--- a/lib/main-app.js
+++ b/lib/main-app.js
@@ -5,24 +5,23 @@ const ipc = electron.ipcMain
 const GhReleases = require('electron-gh-releases')
 const { isPackaged } = app
 // electron.crashReporter.start()
+const singleInstance = app.requestSingleInstanceLock()
 
 var ipcServer = null
 
 var mainWindow = null
 
-var shouldQuit = app.makeSingleInstance(function (commandLine, workingDirectory) {
-  if (mainWindow) {
-    if (process.platform === 'win32') {
-      mainWindow.minimize()
-      mainWindow.restore()
-    }
-    mainWindow.focus()
-  }
-  return true
-})
-
-if (shouldQuit) {
+// Single Instance Lock
+if (!singleInstance) {
   app.quit()
+} else {
+  app.on('second-instance', () => {
+    // Someone tried to run a second instance, it should focus the existing instance.
+    if (mainWindow) {
+      if (!mainWindow.isVisible()) mainWindow.show()
+      mainWindow.focus()
+    }
+  })
 }
 
 var isUpdateReady = false

